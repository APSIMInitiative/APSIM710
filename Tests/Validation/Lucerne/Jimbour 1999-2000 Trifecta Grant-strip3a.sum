     ###     ######     #####   #   #     #   
    #   #    #     #   #        #   ##   ##   
   #     #   #     #   #        #   ##   ##   
   #######   ######     #####   #   # # # #   
   #     #   #              #   #   #  #  #   
   #     #   #         #####    #   #  #  #   
                                              
                                              
 The Agricultural Production Systems Simulator
             Copyright(c) APSRU               

Version                = 7.5
Title                  = Jimbour 1999-2000 Trifecta Grant-strip3a
   Component                        "clock" = %apsim%\Model\Clock.dll
   Component                          "met" = %apsim%\Model\Input.dll
Paddock:
   Component                   "Outputfile" = %apsim%\Model\Report.dll
   Component                        "accum" = %apsim%\Model\Accum.dll
   Component                   "operations" = %apsim%\Model\Operations.dll
   Component                   "fertiliser" = %apsim%\Model\Fertiliser.dll
   Component                    "Grant man" = %apsim%\Model\Manager.dll
   Component                  "Grant Water" = %apsim%\Model\SoilWat.dll
   Component       "Surface Organic Matter" = %apsim%\Model\SurfaceOM.dll
   Component               "Grant Nitrogen" = %apsim%\Model\SoilN.dll
   Component                      "lucerne" = %apsim%\Model\Plant.dll

------- clock Initialisation --------------------------------------------------
     Sequencer phases:
        prepare
        init_step
        do_management
        run_one_step
        do_pasture_water
        process
        do_pasture_growth
        do_stock
        post
        end_step
     Simulation start date = 21/06/1999
     Simulation end date   = 16/05/2000
     Time step =           = 1440 (mins)

------- met Initialisation ----------------------------------------------------
     Sparse data is not allowed
     INPUT File name: grant.met

------- Outputfile Initialisation ---------------------------------------------
     Output frequency:
        post
     Output variables:
        site
        season
        treat
        dd/mm/yyyy as Date
        year
        day
        daysafteremergence
        LAI
        Biomass
        LeafLiveWt
        leaftotalwt
        StemLiveWt
        stemtotalwt
        AboveGroundN
        RootLiveWt
        roottotalwt
        NodeNumber
        sw_dep() as total_sw
        no3() as total_no3
        n_conc_stover
        Height
        WaterStressPhoto
        NStressPhoto

     Output file = Jimbour 1999-2000 Trifecta Grant-strip3a.out
     Format = normal

------- accum Initialisation --------------------------------------------------
     Initialising

------- operations Initialisation ---------------------------------------------
     Operations Schedule
     ===================
       6/08/1999  Prepare      'Surface Organic Matter' tillage type = planter
       6/08/1999  Prepare      lucerne sow plants = 50, sowing_depth = 10, cultivar = trifecta
       6/08/1999  Prepare      fertiliser apply amount = 4 (kg/ha), depth = 50 (mm), type = nh4_n

------- Grant man Initialisation ----------------------------------------------
     Manager rules:
     
     SECTION:- init
     reduce_status = 0
     
     SECTION:- start_of_day
     if (crop_class = 'regrowth' and day = 120 and reduce_status = 0) then
           lucerne reduce
           reduce_status = 1
     endif
     if (day = 150 and crop_class = 'reduced') then
          lucerne spring
          reduce_status = 0
     endif
     site='jimbour'
     season='1999-2000'
     treat=3
     
     SECTION:- end_of_day
     END of rules
     
     Manager creating a new local real variable : reduce_status = 0.00000

------- Grant Water Initialisation --------------------------------------------
     
         - Reading constants
     
        - Reading Soil Property Parameters
     
        - Reading Soil Profile Parameters
     Initial soilwater distributed using "sw" parameter.
     
     
     
                      Soil Profile Properties
        ---------------------------------------------------------------------
              Depth  Air_Dry  LL15   Dul    Sat     Sw     BD   Runoff  SWCON
                mm     mm/mm  mm/mm  mm/mm  mm/mm  mm/mm  g/cc    wf
        ---------------------------------------------------------------------
            0.-  150.  0.150  0.270  0.540  0.580  0.462  1.020  0.762  0.200
          150.-  300.  0.220  0.270  0.530  0.580  0.503  1.030  0.190  0.200
          300.-  600.  0.220  0.280  0.540  0.590  0.457  1.020  0.048  0.200
          600.-  900.  0.220  0.260  0.540  0.580  0.494  1.020  0.000  0.200
          900.- 1200.  0.220  0.310  0.520  0.570  0.483  1.060  0.000  0.200
         1200.- 1500.  0.220  0.320  0.500  0.550  0.496  1.110  0.000  0.200
         1500.- 1800.  0.220  0.330  0.500  0.550  0.467  1.120  0.000  0.200
         1800.- 2100.  0.220  0.330  0.480  0.530  0.449  1.150  0.000  0.200
         2100.- 2400.  0.220  0.340  0.470  0.520  0.466  1.180  0.000  0.200
         2400.- 2700.  0.220  0.330  0.460  0.510  0.469  1.200  0.000  0.200
         2700.- 3000.  0.220  0.360  0.460  0.490  0.476  1.250  0.000  0.200
        ---------------------------------------------------------------------
     
     
     
                  Soil Water Holding Capacity
          ---------------------------------------------------------
              Depth    Unavailable Available  Max Avail.  Drainable
                          (LL15)   (SW-LL15)  (DUL-LL15)  (SAT-DUL)
                            mm        mm          mm         mm
          ---------------------------------------------------------
            0.-  150.      40.50      28.80      40.50       6.00
          150.-  300.      40.50      34.95      39.00       7.50
          300.-  600.      84.00      53.10      78.00      15.00
          600.-  900.      78.00      70.20      84.00      12.00
          900.- 1200.      93.00      51.90      63.00      15.00
         1200.- 1500.      96.00      52.80      54.00      15.00
         1500.- 1800.      99.00      41.10      51.00      15.00
         1800.- 2100.      99.00      35.70      45.00      15.00
         2100.- 2400.     102.00      37.80      39.00      15.00
         2400.- 2700.      99.00      41.70      39.00      15.00
         2700.- 3000.     108.00      34.80      30.00       9.00
          ---------------------------------------------------------
               Totals     939.00     482.85     562.50     139.50
          ---------------------------------------------------------
     
     
     
     
     
     
                  Initial Soil Parameters
          ---------------------------------------------------------
                 Insoil        Salb     Dif_Con   Dif_Slope
          ---------------------------------------------------------
                   0.00        0.13       40.00       16.00
          ---------------------------------------------------------
     
     
     
                  Runoff is predicted using scs curve number:
                Cn2  Cn_Red  Cn_Cov   H_Eff_Depth
                                           mm
          ---------------------------------------------------------
              70.00   20.00    0.80  450.00
          ---------------------------------------------------------
     
     
     
           Using Ritchie evaporation model
            Cuml evap (U):            6.00 (mm^0.5)
            CONA:                     4.00 ()
            Eo from priestly-taylor

------- Surface Organic Matter Initialisation ---------------------------------
     
         - Reading constants
     
         - Reading parameters
     
     
     
                         Initial Surface Organic Matter Data
         ----------------------------------------------------------------------
            Name   Type        Dry matter   C        N        P    Cover  Standing_fr
                                (kg/ha)  (kg/ha)  (kg/ha)  (kg/ha) (0-1)     (0-1)
         ----------------------------------------------------------------------
          wheat     wheat          200.0    80.0     0.8     0.0   0.049     1.0
         ----------------------------------------------------------------------
     
                      Effective Cover from Surface Materials =   0.0
     

------- Grant Nitrogen Initialisation -----------------------------------------
     
        - Reading Parameters
     
        - Reading Constants
     
     
     
           TAV and AMP supplied externally
     
     
     
                      Soil Profile Properties
          ------------------------------------------------
           Layer    pH    OC     NO3     NH4    Urea
                         (%) (kg/ha) (kg/ha) (kg/ha)
          ------------------------------------------------
             1     8.40  1.04   32.59    0.46    0.00
             2     8.70  0.90   34.30    0.46    0.00
             3     8.90  0.89   62.12    0.92    0.00
             4     9.00  0.89   48.65    0.92    0.00
             5     9.00  0.77   26.08    0.95    0.00
             6     9.00  0.45    8.66    1.00    0.00
             7     9.00  0.27    3.36    1.01    0.00
             8     9.00  0.22    3.11    1.04    0.00
             9     8.90  0.16    2.83    1.06    0.00
            10     8.90  0.13    3.24    1.08    0.00
            11     8.90  0.12    3.75    1.13    0.00
          ------------------------------------------------
           Totals              228.68   10.02    0.00
          ------------------------------------------------
     
     
     
     
     
     
                  Initial Soil Organic Matter Status
          ---------------------------------------------------------
           Layer      Hum-C   Hum-N  Biom-C  Biom-N   FOM-C   FOM-N
                    (kg/ha) (kg/ha) (kg/ha) (kg/ha) (kg/ha) (kg/ha)
          ---------------------------------------------------------
             1      15633.9  1116.7   278.1    34.8     9.6     0.2
             2      13735.4   981.1   169.6    21.2     8.3     0.2
             3      27133.4  1938.1   100.6    12.6     6.1     0.2
             4      27207.0  1943.4    27.0     3.4     4.5     0.1
             5      24473.9  1748.1    12.1     1.5     3.4     0.1
             6      14983.5  1070.3     1.5     0.2     2.5     0.1
             7       9071.1   647.9     0.9     0.1     1.8     0.0
             8       7589.2   542.1     0.8     0.1     1.4     0.0
             9       5663.4   404.5     0.6     0.1     1.0     0.0
            10       4679.5   334.3     0.5     0.1     0.8     0.0
            11       4499.6   321.4     0.4     0.1     0.6     0.0
          ---------------------------------------------------------
           Totals  154670.0 11047.9   592.0    74.0    40.0     1.0
          ---------------------------------------------------------
------- Start of simulation  --------------------------------------------------
     Manager creating a new local string variable : season = 1999-2000
     Manager creating a new local real variable : treat = 3.00000
6 August 1999(Day of year=218), operations: 
      Sending tillage message to surface organic matter
6 August 1999(Day of year=218), Surface Organic Matter: 
     
         - Reading residue tillage info
     Residue removed using planter
                                             Fraction Incorporated =     0.10
                                             Incorporated Depth    =    50.00
      Sending sow message to lucerne
6 August 1999(Day of year=218), lucerne: 
     Crop Sow
6 August 1999(Day of year=218): 
     !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
                      APSIM  Fatal  Error
                      -------------------
     plant density ('Population') not specified
     Component name: lucerne
     !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
     
     
      Sending apply message to fertiliser
6 August 1999(Day of year=218), fertiliser: 
     
   - Reading Fertiliser Type Parameters
         4.00 of N as ammonium (nh4_n)
                                         added at depth   50.00 (layer   1)

