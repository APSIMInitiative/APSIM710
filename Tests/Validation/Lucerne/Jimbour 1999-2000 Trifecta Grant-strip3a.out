ApsimVersion = 7.5
Title = Jimbour 1999-2000 Trifecta Grant-strip3a
           site         season          treat           Date           year            day daysafteremergence            LAI        Biomass     LeafLiveWt    leaftotalwt     StemLiveWt    stemtotalwt   AboveGroundN     RootLiveWt    roottotalwt     NodeNumber       total_sw      total_no3  n_conc_stover         Height WaterStressPhoto   NStressPhoto
             ()             ()             ()   (dd/mm/yyyy)         (year)          (day)              (day)      (m^2/m^2)        (kg/ha)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)  (nodes/plant)           (mm)        (kg/ha)            (%)           (mm)               ()             ()
        jimbour      1999-2000          3.000     21/06/1999           1999            172                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1419.431          228.9          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     22/06/1999           1999            173                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1417.373          229.1          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     23/06/1999           1999            174                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1415.576          229.3          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     24/06/1999           1999            175                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1418.016          229.5          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     25/06/1999           1999            176                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1414.400          229.8          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     26/06/1999           1999            177                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1412.845          230.1          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     27/06/1999           1999            178                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1411.599          230.3          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     28/06/1999           1999            179                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1411.035          230.5          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     29/06/1999           1999            180                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1409.978          230.7          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     30/06/1999           1999            181                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1409.016          230.9          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     01/07/1999           1999            182                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1431.748          231.1          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     02/07/1999           1999            183                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1430.161          231.3          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     03/07/1999           1999            184                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1427.506          231.5          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     04/07/1999           1999            185                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1425.622          231.7          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     05/07/1999           1999            186                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1423.470          231.8          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     06/07/1999           1999            187                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1420.999          232.0          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     07/07/1999           1999            188                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1419.680          232.2          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     08/07/1999           1999            189                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1418.562          232.5          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     09/07/1999           1999            190                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1417.576          232.7          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     10/07/1999           1999            191                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1416.793          232.9          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     11/07/1999           1999            192                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1415.971          233.1          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     12/07/1999           1999            193                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1415.209          233.3          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     13/07/1999           1999            194                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1417.263          233.5          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     14/07/1999           1999            195                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1415.204          233.7          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     15/07/1999           1999            196                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1415.442          233.8          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     16/07/1999           1999            197                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1418.702          234.0          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     17/07/1999           1999            198                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1421.469          234.2          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     18/07/1999           1999            199                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1419.256          234.3          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     19/07/1999           1999            200                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1416.418          234.5          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     20/07/1999           1999            201                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1415.641          234.6          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     21/07/1999           1999            202                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1414.916          234.8          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     22/07/1999           1999            203                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1414.235          234.9          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     23/07/1999           1999            204                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1427.015          235.1          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     24/07/1999           1999            205                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1440.242          235.0          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     25/07/1999           1999            206                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1438.121          235.1          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     26/07/1999           1999            207                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1435.246          235.2          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     27/07/1999           1999            208                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1433.872          235.4          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     28/07/1999           1999            209                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1431.083          235.5          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     29/07/1999           1999            210                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1429.106          235.6          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     30/07/1999           1999            211                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1427.849          235.8          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     31/07/1999           1999            212                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1426.786          235.9          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     01/08/1999           1999            213                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1425.847          236.1          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     02/08/1999           1999            214                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1424.998          236.2          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     03/08/1999           1999            215                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1424.216          236.4          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     04/08/1999           1999            216                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1423.488          236.5          0.000          0.000            0.000          0.000
        jimbour      1999-2000          3.000     05/08/1999           1999            217                  0          0.000            0.0           0.00           0.00           0.00           0.00           0.00           0.00           0.00          0.000       1422.803          236.7          0.000          0.000            0.000          0.000
